src/Mod/TechDraw/Gui/QGCustomText.h

```diff
@@ -79,6 +83,7 @@ class TechDrawGuiExport QGCustomText : public QGraphicsTextItem
     Base::Reference<ParameterGrp> getParmGroup(void);
 
     bool isHighlighted;
+    bool tightBounding;
```

Comment:
A comment about what this means/is used for might be helpful to future developers, especially since it's not a verb.

Reply:
Good idea, as that's clearly not an obvious boolean variable. I added a comment there:
`    bool tightBounding;  // Option to use tighter boundingRect() than the standard for plaintext QGCustomText`